ibm/mas_devops/roles/ocp_provision/templates/ipi/install-config.yaml.j2

```diff
@@ -13,6 +13,8 @@ controlPlane:
   platform:
     {{ ipi_platform }}:
       type: {{ ipi_controlplane_type }}
+      rootVolume:
+        size: {{ ipi_storage_size }}
```

Comment:
we are saying here that all IPI installation will use ephemeral storage of 300... shouldn't we, instead, surround it with a conditional to undertand if we are using SNO - similar with [this](https://github.ibm.com/maximoappsuite/ansible-fvt/pull/241/files#r7767476). Criteria could be the same used in CLI to identify we are provisioning SNO, Let's discuss.

Reply:
@alequint how about we leverage this variable that I found used in cli?

Reply:
@Aymen-Ben-S pls look at https://github.ibm.com/maximoappsuite/ansible-fvt/pull/241#discussion_r7775965

Reply:
Please don't introduce sno mode specific _anything_ ... ipi_storage_size is something that is valid sno or not sno ... sno is still just openshift.

` if ipi_storage_size is defined and not ""` is sufficient, there's no reason to limit support for this to sno only.  If someone wants to customize this field .. SNO or non-SNO they can set IPI_STORAGE_SIZE